Tests/SourceFiles/Expected/TestedProtocol.swift

```diff
@@ -23,58 +23,62 @@ class MockTestedProtocol: TestedProtocol, Cuckoo.Mock {
         }
         set {
-            manager.setter("readWriteProperty", value: newValue, original: { self.observed?.readWriteProperty = $0 })(newValue)
+            manager.setter("readWriteProperty", value: newValue, original: { self.observed?.readWriteProperty = $0 })
         }
     }
     
     var optionalProperty: Int? {
         get {
-            return manager.getter("optionalProperty", original: observed.map { o in return { () -> Int? in o.optionalProperty } })()
+            return manager.getter("optionalProperty", original: observed.map { o in return { () -> Int? in o.optionalProperty } })
         }
         set {
-            manager.setter("optionalProperty", value: newValue, original: { self.observed?.optionalProperty = $0 })(newValue)
+            manager.setter("optionalProperty", value: newValue, original: { self.observed?.optionalProperty = $0 })
         }
     }
     
     func noParameter() -> Void {
-        return manager.call("noParameter() -> Void", parameters: Void(), original: observed.map { o in return { () -> Void in o.noParameter() } })()
+        return manager.call("noParameter() -> Void", parameters: Void(), original: observed.map { o in return { () -> Void in o.noParameter() } })
     }
     
     func countCharacters(test: String)-> Int {
-        return manager.call("countCharacters(_:String)-> Int", parameters: (test), original: observed.map { o in return { (test: String)-> Int in o.countCharacters(test) } })(test)
+        return manager.call("countCharacters(_:String)-> Int", parameters: (test), original: observed.map { o in return { (test: String)-> Int in o.countCharacters(test) } })
     }
     
     func withReturn()-> String {
-        return manager.call("withReturn()-> String", parameters: Void(), original: observed.map { o in return { ()-> String in o.withReturn() } })()
+        return manager.call("withReturn()-> String", parameters: Void(), original: observed.map { o in return { ()-> String in o.withReturn() } })
     }
     
     func withThrows() throws -> Void {
-        return try manager.callThrows("withThrows() throws -> Void", parameters: Void(), original: observed.map { o in return { () throws -> Void in try o.withThrows() } })()
+        return try manager.callThrows("withThrows() throws -> Void", parameters: Void(), original: observed.map { o in return { () throws -> Void in try o.withThrows() } })
     }
     
     func withClosure(closure: String -> Int) -> Void {
-        return manager.call("withClosure(_:String -> Int) -> Void", parameters: (closure), original: observed.map { o in return { (closure: String -> Int) -> Void in o.withClosure(closure) } })(closure)
+        return manager.call("withClosure(_:String -> Int) -> Void", parameters: (closure), original: observed.map { o in return { (closure: String -> Int) -> Void in o.withClosure(closure) } })
```

Comment:
Why did you remove the call at the end? I know it might not be useful now that we removed support for @noescape closures, but once Swift 3 is out, we might need to roll it back in.